database/query_builder_test.go

```diff
@@ -0,0 +1,174 @@
+package database
+
+import (
+	"github.com/creasty/defaults"
+	"github.com/icinga/icinga-go-library/logging"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	"testing"
+	"time"
+)
+
+func TestQueryBuilder(t *testing.T) {
+	t.Parallel()
+
+	c := &Config{}
```

Comment:
I'd also like to see tests for PostgreSQL as there are some differences which should at least be tested.